### PR TITLE
London Class 7 - Adem Alagoz - Git&GitHub - Week 1

### PR DESCRIPTION
**Pull request by ademalagoz**
# Your Details

- Your Name: Adem Alagoz
- Your City: London
- Your Slack Name:ademalagoz

# Homework Details

- Module:Git&GitHub 
- Week:1

### CODE REVIEW

**Comment by bisioolajide** [comment]
Great job Adem, I would just make a couple of suggestions:

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-To record what changes you made to the code
+communicate context about a change to other developers working on that project
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+https://git-scm.com/book/en/v2/Getting-Started-Getting-Help
```
Comment (bisioolajide) [inline homework.md:114]: I think we are supposed to include the "Git Cheatsheet" in this answer, but good call on the Git help link.

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
-You delete it
+You can join and prepare to download any programme without any changes in main project
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+You can download any programme to your local computer without any changes in main project
```
Comment (bisioolajide) [inline homework.md:67]: I'm not sure about this answer, I think it should be that the repository is copied to your local drive.

File: homework.md [hunk 3]
```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+After the changes you can request a permission to accept these changes from the owner of the programme(master)
```
Comment (bisioolajide) [inline homework.md:73]: I'm not sure specifying as "the owner of the programme" is correct Adem, you might want re-check, just to avoid confusion.

**Comment by bisioolajide** [comment]
Great Job Adem, I just made a couple of observations you might want to review.

**Comment by ademalagoz** [comment]
Thanks for your help Bisi. I will check and fix them.

**Pull request closed without merging**